src/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,29 @@
-@import 'tailwindcss/base';
-@import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
could we move the tailwind-specific lines to a separate file `base/tailwind.scss` and import it in `base`? The idea is to keep `application.scss` clear and concise.

Reply:
@tyrro resolved in [1aa1c5e](https://github.com/manh-t/react-survey/pull/42/commits/1aa1c5e5c8d8d2c9cda3ac696056fd9e24c867d1)